app/src/org/odk/collect/android/views/ResizingImageView.java

```diff
@@ -157,41 +158,22 @@ public void setFullScreen(){
         }
     }
 
-    public Pair<Integer,Integer> getWidthHeight(int widthMeasureSpec, int heightMeasureSpec, Drawable drawable, double scaleFactor){
+    public Pair<Integer,Integer> getWidthHeight(int widthMeasureSpec, int heightMeasureSpec, double scaleFactor){
 
         int wMode = MeasureSpec.getMode(widthMeasureSpec);
         int hMode = MeasureSpec.getMode(heightMeasureSpec);
```

Comment:
wMode and hMode variables are only read once, so you should be fine not creating the variables at all, but just using the getMode expressions in the conditionals below.